rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigTest.java

```diff
@@ -60,6 +60,7 @@
             "    rskip126: wasabi100",
             "    rskip132: wasabi100",
             "    rskip136: bahamas",
+            "    rskip137: bahamas",
```

Comment:
I'm not quite sure this change is needed, anyway, _rskip137_ shouldn't belong to **bahamas**